### PR TITLE
kubeadm: update the upgrade documentation for 1.17

### PR DESCRIPTION
**Pull request by neolit123**
/sig cluster-lifecycle
/priority important-soon
/assign @fabriziopandini 

fixes https://github.com/kubernetes/website/issues/17582

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md [hunk 1]
```diff
@@ -8,11 +8,12 @@ content_template: templates/task
...
-1.15.x to version 1.16.x, and from version 1.16.x to 1.16.y (where `y > x`).
+1.16.x to version 1.17.x, and from version 1.17.x to 1.17.y (where `y > x`).
 
 To see information about upgrading clusters created using older versions of kubeadm,
 please refer to following pages instead:
 
+- [Upgrading kubeadm cluster from 1.15 to 1.16](https://v1-16.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/)
 - [Upgrading kubeadm cluster from 1.14 to 1.15](https://v1-15.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-15/)
 - [Upgrading kubeadm cluster from 1.13 to 1.14](https://v1-15.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14/)
 - [Upgrading kubeadm cluster from 1.12 to 1.13](https://v1-15.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-13/) or [Upgrading kubeadm HA clusters from v1.12 to v1.13](https://v1-15.docs.kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-ha-1-13/)
```
Comment (neolit123) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md]: 12->13 and 13->14 are outside of the support skew with the release of 1.17.
keeping them here is to help the users, but not ideal.

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md [hunk 2]
```diff
@@ -240,7 +241,7 @@ The upgrade workflow at high level is the following:
 1.  Uncordon the control plane node
 
     ```shell
-    kubectl uncordon $MASTER
+    kubectl uncordon $CP_NODE
```
Comment (neolit123) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md]: renamed MASTER in a couple of places.

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md (hunk 1 above)
Comment (fabriziopandini) [reply content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md]: I think that keeping one more than the official version skew is ok, but after that the user can simply switch to older version of the docs site

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md [hunk 3]
```diff
@@ -109,26 +110,26 @@ The upgrade workflow at high level is the following:
...
-    Controller Manager   v1.15.2   v1.16.0
-    Scheduler            v1.15.2   v1.16.0
-    Kube Proxy           v1.15.2   v1.16.0
-    CoreDNS              1.3.1     1.6.2
-    Etcd                 3.3.10    3.3.15
+    API Server           v1.16.0   v1.17.0
+    Controller Manager   v1.16.0   v1.17.0
+    Scheduler            v1.16.0   v1.17.0
+    Kube Proxy           v1.16.0   v1.17.0
+    CoreDNS              1.6.2     1.6.2
```
Comment (fabriziopandini) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md]: If I remember well we there will be a CoreDNS  bump this cycle.... or I'm wrong?

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md (hunk 2 above)
Comment (fabriziopandini) [reply content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md]: +1

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md (hunk 3 above)
Comment (neolit123) [reply content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md]: this change is not yet in 1.17.0 i will update it.

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md [hunk 4]
```diff
@@ -109,26 +109,26 @@ The upgrade workflow at high level is the following:
...
     Upgrade to the latest version in the v1.16 series:
 
     COMPONENT            CURRENT   AVAILABLE
-    API Server           v1.15.2   v1.16.0
-    Controller Manager   v1.15.2   v1.16.0
-    Scheduler            v1.15.2   v1.16.0
-    Kube Proxy           v1.15.2   v1.16.0
-    CoreDNS              1.3.1     1.6.2
-    Etcd                 3.3.10    3.3.15
+    API Server           v1.16.0   v1.17.0
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md]: nit: `v1.16.3` maybe?

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md [hunk 5]
```diff
@@ -157,9 +157,9 @@ The upgrade workflow at high level is the following:
     [upgrade/config] Making sure the configuration is correct:
     [upgrade/config] Reading configuration from the cluster...
     [upgrade/config] FYI: You can look at this config file with 'kubectl -n kube-system get cm kubeadm-config -oyaml'
-    [upgrade/version] You have chosen to change the cluster version to "v1.16.0"
-    [upgrade/versions] Cluster version: v1.15.2
-    [upgrade/versions] kubeadm version: v1.16.0
+    [upgrade/version] You have chosen to change the cluster version to "v1.17.0"
+    [upgrade/versions] Cluster version: v1.16.0
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md]: nit: `v1.16.3` maybe?

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md (hunk 4 above)
Comment (neolit123) [reply content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md]: 1.16.4 will be out soon.
these are just examples.

**Pull request merged**